frontend/src/metabase/public/components/widgets/EmbedModalContent.jsx

```diff
@@ -111,9 +112,15 @@ class EmbedModalContent extends Component {
       embeddingParams,
     );
 
-    return embedType == null ? (
-      <SharingPane {...this.props} onChangeEmbedType={setEmbedType} />
-    ) : embedType === "application" ? (
+    if (embedType == null) {
```

Comment:
The ternary is getting a little much here, so split it out into `if` statements

Reply:
This looks a lot easier to understand 👍